docs(hostRules): document current matchHost behaviour if a port is supplied (#22007)
renovatebot/renovate: docs/usage/configuration-options.md

}
```

To match specific ports you have to add a protocol to `matchHost`:

```json
{
  "hostRules": [
    {
      "matchHost": "https://domain.com:9118",

>       "enabled": false

Reviewer: It's not immediately clear why enabled=false in the config example.

I would expect to see a allowlist approach instead of a limited block list approach?

Author: This is a follow up of the example above. 
The setting could be anything what we support in hostRules.

    }
  ]
}
```

<!-- prettier-ignore -->
!!! warning

>     Using `matchHost` without a protocol behaves the same as if you had set no `matchHost` configuration.

Reviewer: I'm wondering if we should copy/paste this admonition into the `matchHost` config option docs?

Author: I think that would be too verbose

>     Using `matchHost` without a protocol will lead to a behaviour as there is no `matchHost` configuration at all.

Reviewer: ```suggestion
    Using `matchHost` without a protocol behaves the same as if you had set no `matchHost` configuration.
```

I think this is closer to what you really mean? The current text needs work in any case.

Author: For me both sentences have the same meaning. :men-shrugging:

I have no strong opinion regarding this tough.
